ui/apps/platform/src/Containers/MainPage/Body.tsx

```diff
@@ -114,7 +117,7 @@ function Body({ hasReadAccess, isFeatureFlagEnabled }: BodyProps): ReactElement
                     <Route path={violationsPath} component={AsyncViolationsPage} />
                     <Route path={compliancePath} component={AsyncCompliancePage} />
                     <Route path={integrationsPath} component={AsyncIntegrationsPage} />
-                    <Route path={policiesPath} component={AsyncPolicyManagementPage} />
+                    <Route path={policyManagementBasePath} component={AsyncPolicyManagementPage} />
                     <Redirect exact from={deprecatedPoliciesPath} to={policiesPath} />
```

Comment:
This reminds me to ask if you know some sources of inbound links? I might work on that in 72, if it helps.

For example, I know global search will take some effort to change to new path.

Reply:
hm i'm not sure what you mean by sources of inbound links, could you clarify?

Reply:
Links using the classic /main/policies path that need to be replaced with /main/policy-management/policies as a prerequisite to delete the redirect.